handlers/deploy.go

```diff
@@ -187,11 +187,15 @@ func buildEnv(envProcess string, envVars map[string]string) []string {
 	return env
 }
 
-// BuildEncodedAuthConfig for private registry
+// BuildEncodedAuthConfig parses the image name for a repository, user name, and image name
+// If a repository is not included (ie: username/function-name), 'docker.io/' will be appended
```

Comment:
Sorry to be pedantic but given comments are meant to help someone else understand then `appended` should be `prepended`

Reply:
That's a good point! I couldn't think of that word at the time! Good catch :smile: